### PR TITLE
nodle-parachain dictionary

### PR DESCRIPTION
**Pull request by hobbydev71**

### CODE REVIEW

File: nodle-parachain/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "subql-dictionary",
+  "version": "1.0.0",
+  "description": "A SubQuery Dictionary Project that provides increased indexing speed to all projects",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc -b",
```
Comment (stwiname) [inline nodle-parachain/package.json:7]: We're shifting to new build tooling, can you please try this
```suggestion
    "build": "subql build",
```

File: nodle-parachain/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "repository": "github:subquery/subql-dictionary",
+  "files": [
+    "dist",
+    "schema.graphql",
+    "project.yaml"
+  ],
+  "author": "SubQuery Network",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@polkadot/api": "^7",
```
Comment (stwiname) [inline nodle-parachain/package.json:23]:
```suggestion
    "@polkadot/api": "^8",
```

Comment (hobbydev71) [reply nodle-parachain/package.json]: @stwiname Already changed the version.

**Comment by stwiname** [comment]
Is there still a need for https://github.com/subquery/subql-dictionary/tree/main/nodle?

**Comment by aliXsed** [comment]
> Is there still a need for https://github.com/subquery/subql-dictionary/tree/main/nodle?

Yes, please, we may want to setup a subquery server to browse the history of our solo chain.

**Pull request merged**